pages/common/zint.md

```diff
@@ -0,0 +1,12 @@
+# zint
+
+> CLI tool to generate various (bar/QR) codes.
+> More information: <https://www.zint.org.uk/manual/chapter/4>.
+
+- Generate a barcode and save it:
+
+`zint --data {{your_data_in_utf8}} --output {{path/to/file}}`
+
+- Specify a barcode type (list all possible values with `zint --types`) for generation:
```

Comment:
```suggestion
- Specify a barcode type for generation:
```

Since, this page only has 3 examples, we can make this as a separate example.

Reply:
Implemented in 0c38fda and d25bfc3.